launch_testing/test/launch_testing/test_launch_test_runner_validation.py

```diff
@@ -167,3 +167,23 @@ def generate_test_description(my_param):
 
         dut.validate()  # Make sure this passes initial validation (probably redundant with above)
         runs[0].normalized_test_description(ready_fn=lambda: None)
+
+    def test_launch_description_with_ready_action_changed_timeout(self):
```

Comment:
I don't think this test is actually testing the timeout. It only performs validation of the launch description, but does not actually execute anything as far as I can tell.

I see a few other tests that exercise some of the other command line options, maybe we can do something similar for the `--timeout` option:

- https://github.com/ros2/launch/blob/master/launch_testing/test/launch_testing/test_print_arguments.py
- https://github.com/ros2/launch/blob/master/launch_testing/test/launch_testing/test_xml_output.py

Reply:
I see.